tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -183,17 +183,18 @@ def user_resets_user_roles_via_api_if_required(user_emails: list) -> None:
         if user_email not in allowed_users:
             raise AssertionError(f'`User emails` must contain only allowed users: {allowed_users}')
         try:
-            user_ids = [get_user_details_via_api(user_email)['id']]
-        except IndexError:
-            BuiltIn().log(f'User with email {user_email} does not exist')
-
-        try:
-            result = [user_removes_all_release_and_publication_roles_from_user(user_id) for user_id in user_ids]
-            assert(all(result))
+            user_ids = [get_prerelease_user_details_via_api(user_email)['id']]
+            _ = [user_removes_all_release_and_publication_roles_from_user(user_id) for user_id in user_ids]
             BuiltIn().log(f'All userReleaseRoles & userPublicationRoles reset for user: {user_email}')
+        except IndexError:
+            BuiltIn().log(f'User with email {user_email} does not exist in pre-release user list')
```

Comment:
I suppose this log message could get annoying if the list was changed to allow a mix of pre-release and non pre-release users, and might be better if there was just one log message at the end "User with email blah does not exist in user list or pre-release user list", but probably not an issue right now.

Reply:
Cool, yep one to keep an eye out for if that list gets bigger or more users are going to be added to the `allowed_users` list